modules/virt-binding-devices-vfio-driver.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-pci-passthrough.adoc
+
+[id="virt-binding-devices-vfio-driver_{context}"]
+= Binding a PCI device to the VFIO driver
+
+To bind a PCI device to the VFIO driver, find the values for `vendor-ID` and `device-ID` and add them to the `/etc/modprobe.d/vfio.conf` file. Then, configure the `MachineConfig` object file for the VFIO driver and load the new driver.
+
+.Prerequisites
+. The Intel or AMD hardware must be supported by the CPU and enabled in the BIOS settings.
+
+.Procedure
+. To bind a device to the VFIO driver, apply the kernel module to the `MachineConfig` object file. The following example shows the `MachineConfig` object file.
```

Comment:
isn't this line similar to line 8?

Reply:
Yes! I have revamped the introduction to this procedure. Please review.